debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlSnapshotChangeEventSource.java

```diff
@@ -288,6 +288,9 @@ protected void determineSnapshotOffset(RelationalSnapshotContext<MySqlPartition,
                     LOGGER.info("\t using binlog '{}' at position '{}'", binlogFilename, binlogPosition);
                 }
             }
+            else if (!connectorConfig.getSnapshotMode().shouldStream()) {
```

Comment:
What do you think instead of logging here adding a condition to line 263 and avoid doing the query at all?

Reply:
@mfvitale Agree - done!